relaxng/schemas/build.gradle

```diff
@@ -3,12 +3,12 @@ defaultTasks 'schemas', 'test'
 buildscript {
   repositories {
     mavenCentral()
-    maven { url "http://maven.restlet.org" }
+    maven { url "https://maven.restlet.org" }
   }
 
   dependencies {
-    classpath group: 'org.docbook', name: 'docbook-xslt2', version: '2.4.0'
-    classpath group: 'com.xmlcalabash', name: 'xmlcalabash1-gradle', version: '1.3.5'
+    classpath group: 'org.docbook', name: 'docbook-xslt2', version: '2.5.0'
```

Comment:
Why not 2.6, the latest version?

Reply:
Because I have utterly failed to publish 2.5.1 or 2.6.0. I submit them through Sonatype just like before, they close fine, they pass fine, I release them, and they fall into the void. 😢